acceptance-tests/src/test/resources/features/site/apiCatalogue.feature

```diff
@@ -21,8 +21,6 @@ Feature: API Catalogue in Developer hub
         And I should see elements with attributes:
             | text         | class                                                                             |
             | Inpatient    | nhsd-a-tag nhsd-a-tag--bg-light-grey nhsd-!t-margin-top-3 nhsd-!t-margin-bottom-1 |
```

Comment:
I removed demographics from the catalogue filters which is why I removed this case from the test.